backend/events/admin.py

```diff
@@ -1,3 +1,29 @@
+from django import forms
 from django.contrib import admin
 
+from .models import Event, EventRegistration
+
+
 # Register your models here.
+class EventForm(forms.ModelForm):
```

Comment:
I think we should create a separate file for forms `forms.py` in the app dir. this is the practice to follow seperation of concerns design